### DISCUSSION TITLE
CRYSOL version in ATSAS 4.0.1-1

### DISCUSSION THREAD

**Discussion by I-G-E** [biosaxs-com/atsas-community#1960] [Model evaluation and manipulation] [answered]
Hi,

Which version of crysol is used in ATSAS 4.0.1-1. In previous installations of ATSAS, we have been able to use either crysol 2 or crysol 3 (which models water explicitly rather than implicitly). However it seems in the new ATSAS installation, there is only one version of crysol available and it is unclear how this version treats waters.

Thanks.

**Comment by franke-biosaxs** [comment, marked as answer]
```
% bin/crysol -h
[...]
      --shell=<VALUE>        Shell kind, one of 'directional' (default) or 'water'
```

**Reply by I-G-E** [reply]
Thank you - apologies, I did not notice that option.
